package.json

```diff
@@ -19,6 +19,9 @@
     "json:check": "npx ts-node --esm scripts/schema-check-files.ts src/data testdata/valid",
     "json:check:srcdata": "npx ts-node --esm scripts/schema-check-files.ts src/data",
     "json:check:testdata": "npx ts-node --esm scripts/schema-check-files.ts testdata/valid",
+    "json:generate:types": "npm run json:generate:types:metadata && npm run json:generate:types:timeseries",
+    "json:generate:types:metadata": "npx json-schema-to-typescript --bannerComment \"//DO NOT MODIFY BY HAND.\n//use json:generate:types script.\n\" src/schema/pathwayMetadata.v1.json src/types/pathwayMetadata.v1.d.ts && prettier --write src/types/pathwayMetadata.v1.d.ts",
+    "json:generate:types:timeseries": "npx json-schema-to-typescript --bannerComment \"//DO NOT MODIFY BY HAND.\n//use json:generate:types script.\n\" src/schema/pathwayTimeseries.v1.json src/types/pathwayTimeseries.v1.d.ts && prettier --write src/types/pathwayTimeseries.v1.d.ts",
```

Comment:
@jdhoffa I've scoped the `prettier` line here to just the file that's generated.